.pre-commit-config.yaml

```diff
@@ -0,0 +1,7 @@
+repos:
```

Comment:
mmmm interesting. How does this work?

Reply:
I installed pre-commit https://pre-commit.com/https://pre-commit.com/, and ran it's baseline command which did all the auto formatting. It put a hook into my .git/hooks as per the guide. Wanted to try how this works for multiple users, but it should get triggered by any non compliant commit.